install/update_94_95.php

```diff
@@ -1657,6 +1657,25 @@ function update94to95() {
    }
    /** /Update default right assignement rule */
 
+   if (!$DB->fieldExists('glpi_profiles', 'managed_domainrecordtypes')) {
+      $migration->addField(
+         'glpi_profiles',
+         'managed_domainrecordtypes',
+         'text',
+         [
+            'after'     => 'change_status'
+         ]
+      );
+   }
+
+   $DB->update(
```

Comment:
This will fail if field was not existing prior to migration (previous instruction is async).
You should either put it in an `else`, either run it asyn using a `$migration->addPostQuery()`